config/olm/openshift/0.10.0/manifests/dynatrace-oneagent-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+imagePullSecrets:
+- name: redhat-connect
+- name: redhat-connect-sso
```

Comment:
These pull secrets aren't needed in CSVs.

Reply:
Removed